examples/transportation/docusaurus/signposts.yaml

```diff
@@ -0,0 +1,28 @@
+---
+id: overture:transportation:example:simple-signpost-source
+type: Feature
+geometry:
+  type: LineString
+  coordinates:
+    - [-113.57822030759499, 50.01868388494378]
+    - [-113.57831482025354, 50.018860947117304]
+    - [-113.57851814418316, 50.01923724443006]
+properties:
+  theme: transportation
+  type: segment
+  version: 4
+  updateTime: "2023-08-21T16:23:13-06:00"
+  subType: road
+  connectors:
+    - overture:transportation:example:simple-signpost-connector1
+    - overture:transportation:example:simple-signpost-connector2
+  road:
+    signposts:
+      - direction: forward
+        destination:
+          segmentId: overture:transportation:example:simple-signposts-dest
+          connectorId: overture:transportation:example:simple-turn-restriction-connector2
+          signDirection: North
+          signTextType: Branch
+          signText: "M 21"
+          direction: forward
```

Comment:
> If there are 2 twoway segments connected with a connector in the middle of those two segments (this would be a simple "cross" (+) intersection), if we want to model a signpost we need to have the direction information about both of these segments, so that we know at which part of the "from" segment the signpost is visible and also to which part of the "target" segment is the signpost pointing. Does the schema provide support for this?

Agree, as to the first one. (At which part of the "from" segment the sign~~post~~ is visible.) The second one probably isn't necessary because there's already a reference to a connector, which has a physical location (Point geometry).

The schema does support this. We would require geometric scoping of the sign location itself (on the segment where the sign is located).